test_truststore.py

```diff
@@ -140,6 +143,23 @@ def test_trustme_cert(trustme_ca, httpserver):
     assert len(resp.data) > 0
 
 
+def test_trustme_cert_loaded_via_capath(trustme_ca, httpserver):
+    ctx = truststore.SSLContext(ssl.PROTOCOL_TLS_CLIENT)
+    with tempfile.TemporaryDirectory() as capath:
+        with open(f"{capath}/cert.pem", "wb") as certfile:
+            certfile.write(trustme_ca.cert_pem.bytes())
+        cert_hash = X509.from_cryptography(trustme_ca._certificate).subject_name_hash()
```

Comment:
(The hash is not implemented by `cryptography` because it involves some OpenSSL-specific normalization of the ASN.1 subject name -- see https://github.com/pyca/cryptography/issues/6456.)